src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt

```diff
@@ -105,7 +106,13 @@ object VarselServiceSpek : Spek({
                 behandlerInnkallingUuid = null
             )
 
-            verify(exactly = 1) { altinnClient.sendToVirksomhet(virksomhetsbrevId, virksomhetsPdf, VIRKSOMHETSNUMMER_NO_NARMESTELEDER) }
+            val altinnMelding = createAltinnMelding(virksomhetsbrevId, VIRKSOMHETSNUMMER_NO_NARMESTELEDER, virksomhetsPdf, MotedeltakerVarselType.INNKALT)
+
+            verify(exactly = 1) {
+                altinnClient.sendToVirksomhet(
+                    altinnMelding
+                )
+            }
```

Comment:
Dette blir en test AltinnClient inne i VarselServiceSpek, enten så bør dette flyttes til en egen AltinnClientSpek, eller aller helst lage en fullverdig mock av InsertCorrespondenceV2/AltinnClient som nevnt i en kommentar, for da kan man kun kjøre en en verify her og trenger ikke denne linjen:
`val altinnMelding = createAltinnMelding(virksomhetsbrevId, VIRKSOMHETSNUMMER_NO_NARMESTELEDER, virksomhetsPdf, MotedeltakerVarselType.INNKALT)
`